pulp_smash/constants.py

```diff
@@ -12,6 +12,13 @@
     http://pulp.readthedocs.io/en/latest/dev-guide/conventions/sync-v-async.html#call-report
 """
 
+CONTENT_SOURCE_ID = 'content-world'
```

Comment:
@elyezer, actually there's no content source for a fresh Pulp, in `/etc/pulp/content/sources/conf.d/`

@Ichimonji10, as for creating content source, I understand what you mean...but currently I can't find an existing module related to content source, therefore I use a constant as workaround.